core/digital-twin-lib/src/DAppContainer.ts

```diff
@@ -198,17 +220,20 @@ class DAppContainer extends Container {
    * @param      {string}  entriesToLoad  load only specific entriess
    */
   public async loadEntryValues(entriesToLoad?: string[]): Promise<void> {
+    // TODO: why is key for type returned here? For what is it used?
```

Comment:
Type is a general property on every DataContainer. For our case, we can just remove it.